devices/Hx711/Hx711.cs

```diff
@@ -14,7 +14,14 @@ namespace Iot.Device.Hx711
     public class Scale
     {
         // pulse train required to read a sample and setup gain factor for next reading
-        private readonly byte[] _readSamplePulseTrain;
+        private byte[] _readSamplePulseTrain;
```

Comment:
please keep it readonly, as you have no reason to resize this array in your code on what I can understand

Reply:
done